src/main/java/com/pokegoapi/api/map/pokemon/CatchablePokemon.java

```diff
@@ -457,7 +458,7 @@ protected CatchItemResult handle(ByteString result) throws RemoteServerException
 	 * @throws RemoteServerException if the server failed to respond
 	 */
 	public CatchItemResult useItem(ItemId item) throws LoginFailedException, RemoteServerException {
-		return useItemAsync(item).toBlocking();
+		return AsyncHelper.toBlocking(useItemAsync(item));
```

Comment:
How's a transformer going to help in this situation? The goal is to unwrap the Observable and handle all RuntimeExceptions to throw their checked versions. A transformer will return another Observable, which is exactly not what I need or am I missing some other usage of a transformer?